src/instrument/utils/dm_metadata_file.py

```diff
@@ -0,0 +1,140 @@
+"""
+Write the metadata file for processing a bluesky run with a DM workflow.
+
+All metadata content comes from the bluesky run's metadata dictionary. The
+bluesky session is responsible to provide all expected metadata keys.
+"""
+
+import logging
+import pathlib
+
+import databroker
+import h5py
+import yaml
+
+from .conversions import ts2iso
+from .iconfig_loader import iconfig  # noqa
+
+CATALOG = iconfig.get("DATABROKER_CATALOG", "8idi_xpcs")
```

Comment:
Why is 8idi_xpcs hard coded, isn't it already written in the iconfig?

Reply:
From testing

Reply:
sounds good. Guess it's good to be removed then?